Projects/UOContent/Mobiles/Monsters/Humanoid/Magic/EvilMage.cs

```diff
@@ -12,6 +12,7 @@ public EvilMage() : base(AIType.AI_Mage)
             Name = NameList.RandomName("evil mage");
             Title = "the evil mage";
             Body = Core.UOR ? 124 : 0x190;
+            Hue = Core.UOR ? 0 : Race.Human.RandomSkinHue();
```

Comment:
```suggestion
            if (Core.UO)
            {
                Hue = Race.Human.RandomSkinHue();
            }
```

Reply:
Actually, I think EvilMages and EvilMageLords only had the pale models (0x190) and no hue switching.

Reply:
Then let's get rid of the hue change?